packages/block-library/src/heading/editor.scss

```diff
@@ -14,7 +9,9 @@
 // level toolbar.
 .block-library-heading-level-toolbar {
 	border: none;
+
 	.components-toolbar-group {
 		flex-wrap: nowrap;
+		flex-direction: column;
```

Comment:
Can we make this be part of the toolbar set of components?

Reply:
Yeah! That makes more sense.